Kernel/Assertions.h

```diff
@@ -6,11 +6,13 @@
 
 #pragma once
 
+#include <AK/Platform.h>
+
 #define __STRINGIFY_HELPER(x) #x
 #define __STRINGIFY(x) __STRINGIFY_HELPER(x)
 
 [[noreturn]] void __assertion_failed(const char* msg, const char* file, unsigned line, const char* func);
-#define VERIFY(expr) (static_cast<bool>(expr) ? void(0) : __assertion_failed(#expr, __FILE__, __LINE__, __PRETTY_FUNCTION__))
+#define VERIFY(expr) (LIKELY(expr) ? void(0) : __assertion_failed(#expr, __FILE__, __LINE__, __PRETTY_FUNCTION__))
```

Comment:
tbh I'd rather have `__builtin_expect` used directly here
Humans are normally pretty bad with annotating these, and giving them an easy macro tends to shit on perf _and_ readability (that's my experience from C projects, but I imagine the same applies here)